### PR TITLE
Multiple waiting areas

### PR DESCRIPTION
**Pull request by lgarron**
Followup to #601, which should be merged first.

(If we decide not to land #601, I can update this to refer to "Competitors Area".)

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -208,8 +208,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For Solving With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
-- 7h) The competition area must have a Competitors Area.
-    - 7h1) The organization team may require that a competitor who has been called to compete must remain within the Competitors Area until the competitor has finished all attempts for the round.
+- 7h) The competition area must have one or more Competitor Waiting Areas.
+    - 7h1) The organization team may require that a competitor who has been called to compete must remain within an assigned Competitor Waiting Area between attempts, until the competitor has finished all attempts for the round.
```
Comment (AlbertoPdRF) [inline wca-regulations.md]: > ... within **the** assigned ...?

Each competitor will have one and only one assigned waiting area for every round they compete at.

File: wca-regulations.md [hunk 2]
```diff
@@ -369,7 +369,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise, the stopwatch time is the original recorded time.
     - A1c) A competitor participating in an event must be able to fulfill the event's requirements (e.g. know how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result. Penalty: disqualification of the attempt (DNF)  or disqualification from the event (see [Regulation 2j](regulations:regulation:2j)), at the discretion of the WCA Delegate.
 - A2) Scrambling:
-    - A2a) When called for a round, the competitor submits a puzzle, in its solved state, to the scrambler. The competitor then waits in the Competitors Area until they are called to compete.
+    - A2a) When called for a round, the competitor submits a puzzle, in its solved state, to the scrambler. The competitor then waits in a Competitor Waiting Area until they are called to compete.
```
Comment (AlbertoPdRF) [inline wca-regulations.md]: Similar case as above.

**Review by AlbertoPdRF — APPROVED** [review]
Left two comments.

**Comment by lgarron** [comment]
@AlbertoPdRF, are you still okay with the updated changes?

File: wca-guidelines.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ To be more informative, each Guideline is classified using one of the following
 
 - 7d+) [ADDITION] The temperature of the competition area should be 21 to 25 degrees Celsius.
 - 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
-- 7h2+) [ADDITION] The competitors in the Competitors Area should not be able to see the puzzles of the competitors on stage.
+- 7h2+) [ADDITION] Competitors in the Competitors Area who are waiting between attempts should not be able to see the puzzles of the competitors on stage.
```
Comment (AlbertoPdRF) [inline wca-guidelines.md]:
```suggestion
- 7h2+) [ADDITION] Competitors in a Competitors Area who are waiting between attempts should not be able to see the puzzles of the competitors on stage.
```

File: wca-regulations.md [hunk 4]
```diff
@@ -369,7 +369,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise, the stopwatch time is the original recorded time.
     - A1c) A competitor participating in an event must be able to fulfill the event's requirements (e.g. know how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result. Penalty: disqualification of the attempt (DNF)  or disqualification from the event (see [Regulation 2j](regulations:regulation:2j)), at the discretion of the WCA Delegate.
 - A2) Scrambling:
-    - A2a) When called for a round, the competitor submits a puzzle, in its solved state, to the scrambler. The competitor then waits in the Competitors Area until they are called to compete.
+    - A2a) When called for a round, the competitor submits a puzzle, in its solved state, to the scrambler. The competitor then waits in a Competitors Area until they are called to compete.
```
Comment (AlbertoPdRF) [inline wca-regulations.md]:
```suggestion
    - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. The competitor then waits in their assigned Competitors Area until they are called to compete.
```

**Review by AlbertoPdRF — APPROVED** [review]
Small changes suggested!

**Comment by lgarron** [comment]
Thanks!

**Pull request merged**